pkgs/os-specific/linux/criu/default.nix

```diff
@@ -12,8 +12,9 @@ stdenv.mkDerivation rec {
   };
 
   enableParallelBuilding = true;
-  nativeBuildInputs = [ pkgconfig docbook_xsl which makeWrapper docbook_xml_dtd_45 ];
-  buildInputs = [ protobuf protobufc asciidoc xmlto libpaper libnl libcap libnet python iptables ];
+  nativeBuildInputs = [ pkgconfig docbook_xsl which makeWrapper docbook_xml_dtd_45 python3 python3.pkgs.wrapPython ];
+  buildInputs = [ protobuf protobufc asciidoc xmlto libpaper libnl libcap libnet iptables ];
+  propagatedBuildInputs = with python3.pkgs; [ python python3.pkgs.protobuf ];
```

Comment:
Need to use `python3.pkgs.protobuf` instead of `protobuf` here. I realized this after opening the PR (s. description) but seems like I didn't test it properly back then (maybe I did already replace `wrapPythonPrograms` with `wrapPythonProgramsIn ...`).